### ISSUE TITLE
Retrieving more than one relationship

### ISSUE THREAD

**Issue by ratzr15** [wvteijlingen/Spine#190]
I'm trying to get more than one relationship using `, ``include`

As : "`selections.options,images`"

I'm registering the `Query` for the Resource associated and passing the `ResourceID`

But only _one_ of the Resource gets _loaded_.

**_My Query:_**
```
        var query    = Query.init(resourceType: MyResource.self, resourceIDs: ["1111"])
        query.include(selections.options,images)
```

What am i missing here ?

**Comment by markst** [comment]
@ratzr15 are you sure this isn't a server side issue?

**Comment by ratzr15** [comment]
@markst  Nope, I have always had issue accessing more than one relationship. I'm able to load more than one relationship as :`  var query = Query(resourceType: MyResource.self, resourceCollection: resource.respurce_children!)`

But it increases the number of Spine requests for example if have to get `resource1.resource1relation,resource2,resource3` I have to make 3 Spine requests to get the relative data.

Any example on how to load more than one relationship would be really appreciated.

**Comment by ratzr15** [comment]
@markst  Figured it out 🤤

I should be querying relationships as ` [String]`

` query.include(selections.options,images)` should be as `query.include("selections.options","images","children")`
